_about_livecoms/03_earlycareer.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: single
+sidebar:
+  nav: earlycareer.md
+title: For Early Career Scientists 
+excerpt: How early career scientists can get involved with LiveCoMS
+permalink: /about/earlycareer/
+---
+
+We welcome early career scientists (undergraduate students, graduate
+students and postdocs) to help participate in LiveCoMS.  There are two
```

Comment:
"to help participate" should probably be "to participate"

Reply:
Done.